drf_braces/utils.py

```diff
@@ -130,6 +141,23 @@ def get_class_name_with_new_suffix(klass, existing_suffix, new_suffix):
 
 
 def get_attr_from_base_classes(bases, attrs, attr, **kwargs):
+    """
+    The base class attributes are retrieved if they are not already
+    present on the object.
+
+    Args:
+        bases (tuple, list): The base classes for a class
+        attrs (dict): The attributes of the class
+        attr (str): Specific attribute being looked for
+
```

Comment:
you can add explicit `default` arg. I moved it to `**kwargs` so that I can detect when its provided and when its not since `default=None` will mask when people actually want the default to be `None`...